pyeapi/api/interfaces.py

```diff
@@ -587,7 +587,8 @@ def get_members(self, name):
         grpid = re.search(r'(\d+)', name).group()
         command = 'show port-channel %s all-ports' % grpid
         config = self.node.enable(command, 'text')
-        return re.findall(r'Ethernet[\d/]*', config[0]['result']['output'])
+        return re.findall(r'\b(?!Peer)Ethernet[\d/]*\b',
+                          config[0]['result']['output'])
```

Comment:
The regex here looks for word boundaries \b and then does a negative look ahead to make sure Peer isn't matched.